go.mod

```diff
@@ -3,10 +3,10 @@ module github.com/tailscale/tailscale-android
 go 1.23.1
 
 require (
-	github.com/tailscale/wireguard-go v0.0.0-20240905161824-799c1978fafc
+	github.com/tailscale/wireguard-go v0.0.0-20241113014420-4e883d38c8d3
 	golang.org/x/mobile v0.0.0-20240806205939-81131f6468ab
 	inet.af/netaddr v0.0.0-20220617031823-097006376321
-	tailscale.com v1.75.0-pre.0.20241031190034-6985369479db
+	tailscale.com v1.75.0-pre.0.20241119130719-00517c818956
```

Comment:
Despite the 1.75 here, the version in the manifest is the correct thing, and that's the correct hash.